packages/backend/src/app/flags/flag.service.ts

```diff
@@ -14,81 +16,97 @@ export const flagService = {
       value: flag.value,
     });
   },
-  async getOne(flagId: FlagId): Promise<Flag | null> {
+  async getOne(flagId: ApFlagId): Promise<Flag | null> {
     return await flagRepo.findOneBy({
       id: flagId,
     });
   },
   async getAll(): Promise<Flag[]> {
     const flags = await flagRepo.find({});
-
     const now = new Date().toISOString();
     const created = now;
     const updated = now;
-
+    // eslint-disable-next-line @typescript-eslint/no-var-requires
+    const currentVersion = require('../../../../../package.json').version;
```

Comment:
I think it's a bad idea to use `import` and `require` in the same file, btw there's a way to import JSON files using `import`, see https://www.typescriptlang.org/tsconfig#resolveJsonModule